x-pack/plugins/canvas/common/lib/dataurl.js

```diff
@@ -34,7 +34,7 @@ export function parse(str, withData = false) {
   };
 }
 
-export function isValid(str) {
+export function isValidDataUrl(str) {
```

Comment:
Why did you rename this export? Was it required for some reason?

Reply:
same as above